src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,24 @@
+@use '../utils/variables' as *;
+
+.stars {
+  display: flex;
+  align-items: center;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    padding-right: 4px;
+    background-size: 13px;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-image: url(/src/images/star.svg);
+  }
+
+  &--1 &__star:nth-child(-n + 1),
+  &--2 &__star:nth-child(-n + 2),
+  &--3 &__star:nth-child(-n + 3),
+  &--4 &__star:nth-child(-n + 4),
+  &--5 &__star:nth-child(-n + 5) {
+    background-image: url(/src/images/star-active.svg);
```

Comment:
Similar to the previous comment, the path to 'star-active.svg' should be relative to the CSS file. Use '../images/star-active.svg' to correct the path.